pkg/gomod/bytes/parser_test.go

```diff
@@ -0,0 +1,33 @@
+package bytes
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	parser "github.com/gomods/athens/pkg/gomod"
+)
+
+func TestContentParser_ModuleName(t *testing.T) {
+	a := assert.New(t)
+
+	var testCases = []struct {
+		content     []byte
+		expected    string
+		expectedErr error
+	}{
+		{[]byte(`module "my/thing"`), "my/thing", nil},
+		{[]byte(`module "my/thing2`), "", parser.ErrNotFound},
+		{[]byte(`foobar`), "", parser.ErrNotFound},
```

Comment:
try moultiple modules in one file, first should be selected, don't know how vgo behaves in this case

Reply:
Thanks! Added in https://github.com/gomods/athens/pull/72/files#diff-6d0b6ed72e589e371bc00e27c79d5d9cR28